src/app/about/about.component.spec.ts

```diff
@@ -20,6 +20,7 @@ import { AdvancedsearchComponent } from '../advancedsearch/advancedsearch.compon
 import { SearchBarComponent } from '../search-bar/search-bar.component';
 import { ContactComponent } from '../contact/contact.component';
 import { ModalComponent } from 'ng2-bs3-modal/ng2-bs3-modal';
+import { TopbarComponent } from '../topbar/topbar.component';
```

Comment:
@nikhilrayaprolu The code which is written in original repository, applying some changes into it was not giving out good results. So, I followed the loklak in which they, have created a separate component for it and it become easy to customize it.